### ISSUE TITLE
Refactor Contact form

### ISSUE THREAD

**Issue by Kara-Zor-El** [Trent-Computer-Science-Club-Association/ClubWebsite#32]
The contact form is designed in a way where it is extensible and such but the code currently is messy. Needs fixing. Something like:
```
FormTypes
FormConstructionFunctions
Break Somehow (use comments)
Everything todo with the actual component
```

**Comment by spotandjake** [comment]
The contact form implemented by #28 is currently built in an extensible way unfortuntatly because of the needs of our use case this results in a really hard to understand module. It might make more sense to use a standard ui component or build a specalized form. The current implementation is not bad it is just hard to follow and hard to maintain.
